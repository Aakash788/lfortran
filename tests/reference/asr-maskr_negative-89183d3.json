{
    "basename": "asr-maskr_negative-89183d3",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/maskr_negative.f90",
    "infile_hash": "002bf5153bb072319faf7015c970d9d08af84a968ff7683136dd2d3e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-maskr_negative-89183d3.stderr",
    "stderr_hash": "dda6eb2d1243fdc81503d5ed42a793d54435c5708ca002623de1548f",
    "returncode": 2
}