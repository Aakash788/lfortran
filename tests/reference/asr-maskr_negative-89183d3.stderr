semantic error: `I` must be nonnegative
 --> tests/errors/maskr_negative.f90:2:12
  |
2 |    print*, maskr(-24)
  |            ^^^^^^^^^^ 
